### PR TITLE
Create 2022-11-22-featureprobe.md

### PR DESCRIPTION
**Pull request by nanxijms**

### CODE REVIEW

File: blog/2022-11-22 featureprobe-md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+slug | title
```
Comment (frankzhaopeng) [inline blog/2022-11-22 featureprobe-md:2]: change the slug to a unique title

File: blog/2022-11-22 featureprobe-md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+-- | --
+FeatureProbe Rust | 用 Rust 开发跨平台 SDK 探索和实践
```
Comment (frankzhaopeng) [inline blog/2022-11-22 featureprobe-md:4]: Is this title right?

**Comment by frankzhaopeng** [comment]
An Chinese version is needed as well in folder: /i18n/zh-CN/docusaurus-plugin-content-blog

File: blog/2022-11-22 featureprobe-md (hunk 2 above)
Comment (nanxijms) [reply blog/2022-11-22 featureprobe-md:4]: Not at all, I copied the last article......

**Pull request merged**